dist/pie-chart.js

```diff
@@ -178,12 +202,13 @@ getTools: function(dimensions, options) {
   }
 
   return {pie: pieLayout, arc: arc};
-},
+}
+,
 
 addLegend: function(svg) {
   var items = svg.append("g")
     .attr("id", "n3-pie-legend");
-
+  
```

Comment:
Do you have `trim_trailing_white_space_on_save` option set to `true` in your Sublime?